lithops/standalone/standalone.py

```diff
@@ -198,6 +204,54 @@ def create_workers(workers_to_create):
 
             return list(new_workers)
 
+        def wait_workers_ready(workers_num):
+            logger.info(f'Waiting {workers_num} workers to become ready')
```

Comment:
There could be multiple lithops clients using the same VM as you told me some days ago.
In this method you check the total_workers are correctly running, but don't check which workers. 
I mean, lets say one client is using 5 workers, and then you invoke a map() that will create 5 new workers (because the other 5 are busy). This method will check that 5 workers are running, but probably it will check the workers that are being used by the other client, not the ones created for running this map(). So it can raise an exception based on the workers used by another client.

Reply:
@JosepSampe Good point, will change it to track created workers set.

Reply:
Fixed, replaced to track new_workers list